src/190B20.c

```diff
@@ -1897,15 +2753,47 @@ void func_80266D6C(ActorPart* part, s32 arg1) {
     }
 }
 
-INCLUDE_ASM(s32, "190B20", func_80266DAC);
+void func_80266DAC(Actor* actor, s32 arg1) {
+    ActorPart* partIt = &actor->partsTable[0];
+
+    while (partIt != NULL) {
+        if (!(partIt->flags & ACTOR_PART_FLAG_INVISIBLE) &&
+           (partIt->idleAnimations != NULL) &&
+           !(partIt->flags & ACTOR_PART_FLAG_2))
+        {
+            func_80266D6C(partIt, arg1);
+        }
+        partIt = partIt->nextPart;
+    }
+}
 
 void func_80266E14(ActorPart* part) {
     if (part->idleAnimations != NULL && !(part->flags & ACTOR_PART_FLAG_2)) {
         part->decorationTable->unk_6C0 = 0;
     }
 }
 
-INCLUDE_ASM(s32, "190B20", func_80266E40);
+void func_80266E40(Actor* actor) {
```

Comment:
Definitely `// TODO: bad match` this func w.r.t. the `e` and `f` and the `do..while(0)`

Reply:
I think the todo: improve match is sufficient since it points us at issues in these functions, but I added it anyway